androidClient/app/src/main/java/com/csatimes/dojma/fragments/VideosFragment.java

```diff
@@ -0,0 +1,78 @@
+package com.csatimes.dojma.fragments;
+
+import android.app.Activity;
+import android.content.Context;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.csatimes.dojma.R;
+import com.csatimes.dojma.adapters.VideosAdapter;
+import com.csatimes.dojma.models.VideosItem;
+import com.facebook.drawee.backends.pipeline.Fresco;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.GridLayoutManager;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import static com.csatimes.dojma.activities.AboutDojmaActivity.TAG;
+import static com.csatimes.dojma.utilities.DHC.TAG_PREFIX;
+import static com.csatimes.dojma.utilities.DHC.getGridSpan;
+
+public class VideosFragment extends Fragment {
+
+    //private static final String TAG = TAG_PREFIX + VideosFragment.class.getSimpleName();
+    private RecyclerView videosRv;
+
+    String videoURL[] = {"https://www.facebook.com/DoPy.BITSGoa/videos/622644644796899/","https://www.facebook.com/bitsquark/videos/309257229884592/","https://www.facebook.com/bitsquark/videos/2300637253556600/","https://www.youtube.com/watch?v=IM2czqavlWM", "https://www.youtube.com/watch?v=737dCFBqO4I", "https://www.youtube.com/watch?v=yuboWIBPDvk", "https://www.youtube.com/watch?v=pXCpc2Bcirs", "https://www.youtube.com/watch?v=4AQm22YQfYU", "https://www.youtube.com/watch?v=3PtbK_0j1sw", "https://www.youtube.com/watch?v=WDIeS7NHTpU", "https://www.youtube.com/watch?v=LvetJ9U_tVY", "https://www.youtube.com/watch?v=6JYIGclVQdw"};
```

Comment:
create a new class (YoutubeVideo, maybe) to hold these data members (url, title, etc)